client.go

```diff
@@ -48,3 +48,10 @@ func (m *ClientMock) hasStub(method string) bool {
 
 	return false
 }
+
+func (m *ClientMock) Reset() {
```

Comment:
I don't see the need for `Reset` in this package. It's best to create a new mock each time you need to set the expectations (for example, per test). If you need to do something more ticky than that you should handle it in your test code.

Reply:
ok, tks